lib/pure/terminal.nim

```diff
@@ -462,15 +509,35 @@ elif not defined(windows):
     result = stdin.readChar()
     discard fd.tcsetattr(TCSADRAIN, addr oldMode)
 
+# Deprecated procs assuming output to stdout:
+template setCursorPos*(x, y: int) {.deprecated.} = setCursorPos(stdout, x, y)
```

Comment:
That's the only thing I'm unhappy with. I don't see the need to deprecate the existing 'setCursorPos' etc operating on stdout.